pkg/build/rpm.go

```diff
@@ -0,0 +1,55 @@
+package build
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+)
+
+func (b *Builder) getRPMFileNames() error {
```

Comment:
This will currently modify the internal state of the builder by storing results which should generally not be a part of it.

A function named `get...`, `retrieve...`, `find...`, etc. should return the value itself (and an error if it's possible to fail). This way we decouple temporary data from the struct state and are able to more easily test it.

Reply:
👍  will update